src/System.Data.Common/ref/System.Data.Common.Manual.cs

```diff
@@ -79,4 +79,8 @@ public static bool CanGetColumnSchema(this DbDataReader reader)
         }
     }
 
+    public partial interface IDbColumnSchemaGenerator
```

Comment:
It doesn't really matter but you can put this in the non-manual.cs file now. At this point none of the CS ref files are generated and thus they are all manual.